### PR TITLE
Match create and addMembers messages against spec

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-tribes2/issues/18

* [x] add for create
* [x] add for addMembers
* [x] include tests from tribes1
* [x] see that tests actually pass once https://github.com/ssbc/ssb-tribes2/pull/20 is merged
* [x] wait for https://github.com/ssbc/private-group-spec/pull/19 to be merged and published and update package.json here

### CODE REVIEW

**Comment by Powersource** [comment]
when manually running the new tests they pass

File: spec/add-member.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2022 Mix Irving
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const Validator = require('is-my-ssb-valid')
+const schema = require('private-group-spec').schema.group.addMember
```
Comment (mixmix) [inline spec/add-member.js:6]: Nice one past mix!
Good idea using these

File: test/spec/add-member.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          '%lm6Sqk3fFJA2EQFiyJ1MSASDASDASDASDASDAS/BZ9E=.sha256',
+          '%Sp294oBk7OJxizvPOlm6Sqk3fFJA2EQFiyJ1MS/BZ9E=.sha256',
+        ],
+      },
+    },
+  }
+  times(15, () => base.recps.push(FeedId()))
+
+  return Object.assign(base, overwrite)
+}
```
Comment (mixmix) [inline test/spec/add-member.test.js:40]: I think these tests are good, but should live next to the schema in `private-group-spec`

**Review by mixmix — CHANGES_REQUESTED** [review]
please move tests of those schemas to `private-group-spec`

**Review by staltz — APPROVED** [review]
I took a QUICK look, I don't know if I agree with everything (I barely looked at tests), but I'm totally okay if you just merge.

PS: in other words, I delegate reviewing to Mix, who knows about the schemas better than I do.

File: test/spec/add-member.test.js (hunk 2 above)
Comment (Powersource) [reply test/spec/add-member.test.js:40]: moved it there in this pr https://github.com/ssbc/private-group-spec/pull/19

File: spec/add-member.js (hunk 1 above)
Comment (Powersource) [reply spec/add-member.js]: yeah nice stuff.

moved this set up of the validator into the spec module as well, needed that to be able to run the tests there. and make for less boilerplate in this module too

File: package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "envelope-spec": "^1.1.0",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
-    "private-group-spec": "^1.1.3",
+    "private-group-spec": "powersource/private-group-spec#spec-tests",
```
Comment (mixmix) [inline package.json:35]: TODO

**Review by mixmix — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Powersource) [reply package.json:35]: @mixmix @staltz need a new minor version of private-group-spec

Comment (staltz) [reply package.json:35]: @Powersource Done, v1.2.0

**Pull request merged**